src/error.rs

```diff
@@ -37,6 +38,8 @@ pub enum Error {
     TooLarge,
     /// An invalid `Status`, such as `1337 ELITE`.
     Status,
+    /// An ivalid header value or header value fragment.
+    Invalid,
```

Comment:
`Header` is already taken and used when header blocks fail to parse. I rename it to `Value`.